### PR TITLE
icons changes for localization and tooltips adding

### PR DESCRIPTION
**Pull request by Sancerio**
hi @Benglin, please review and merge. @elayabharath has approved the icons

### CODE REVIEW

File: src/DynamoCoreWpf/Properties/Resources.en-US.resx [hunk 1]
```diff
@@ -1415,4 +1415,12 @@ Click CANCEL to cancel the update.</value>
   <data name="InstallMessageCaption" xml:space="preserve">
     <value>Install Dynamo</value>
   </data>
+  <data name="DynamoGeometryButtonToolTip" xml:space="preserve">
+    <value>Geometry</value>
+    <comment>Geometry display tooltip</comment>
+  </data>
```
Comment (Benglin) [inline src/DynamoCoreWpf/Properties/Resources.en-US.resx]: This should be renamed to `InCanvasGeomButtonToolTip`, and its content can be rephrased as:

`Enable background 3D preview navigation (Ctrl + G)`

File: src/DynamoCoreWpf/Properties/Resources.en-US.resx [hunk 2]
```diff
@@ -1415,4 +1415,12 @@ Click CANCEL to cancel the update.</value>
...
+  <data name="DynamoNodeButtonToolTip" xml:space="preserve">
+    <value>Node</value>
+    <comment>Node display tooltip</comment>
+  </data>
```
Comment (Benglin) [inline src/DynamoCoreWpf/Properties/Resources.en-US.resx]: This should be renamed to `InCanvasNodeButtonToolTip`, and its content can be rephrased as:

`Enable graph view navigation (Ctrl + G)`

**Comment by Benglin** [comment]
Good first attempt, @Sancerio! Once you've updated those please notify me through my GitHub alias (@Benglin). Thanks!

**Comment by Sancerio** [comment]
Thanks!
Here is the preview of the changes
changes

**Comment by Benglin** [comment]
Thanks @Sancerio, LGTM!

**Pull request merged**